runtime/layouts/microbit_v2.ld

```diff
@@ -0,0 +1,11 @@
+/* Layout for the micro:bit v2 board, used by the examples in this repository. */
+
+MEMORY {
+  FLASH (rx) : ORIGIN = 0x00040000, LENGTH = 256K
+  RAM (rwx) : ORIGIN = 0x20004000, LENGTH = 112K
```

Comment:
Should probably use `(X)` and `(W)` here rather than `(rw)` and `(rwx)` to be consistent with the other layout files in this directory.

Reply:
Thank you for pointing that out, fixed it.